.travis.yml

```diff
@@ -67,27 +73,30 @@ script:
   - pip list
   - which mlflow
   - echo $MLFLOW_HOME
-  - mlflow sagemaker build-and-push-container --no-push --mlflow-home .
+  - SAGEMAKER_OUT=$(mktemp)
+  - if mlflow sagemaker build-and-push-container --no-push --mlflow-home . > $SAGEMAKER_OUT 2>&1; then
+      echo "Sagemaker container build succeeded.";
+    else
+      echo "Sagemaker container build failed, output:";
+      cat $SAGEMAKER_OUT;
+    fi
   - ./lint.sh
   # Run tests that don't leverage specific ML frameworks
   - pytest --cov=mlflow --verbose --large --ignore=tests/h2o --ignore=tests/keras
     --ignore=tests/pytorch --ignore=tests/pyfunc--ignore=tests/sagemaker --ignore=tests/sklearn
     --ignore=tests/spark --ignore=tests/tensorflow
   # Run ML framework tests in their own Python processes. TODO: find a better method of isolating
   # tests.
-  - pytest --cov=mlflow --verbose tests/h2o --large
```

Comment:
We keep the `--cov=mlflow` in the first invocation of `pytest` above. This is enough to run codecov on the source tree rooted at "mlflow", as per the output of `pytest --help`:

```
...
coverage reporting with distributed testing support:
  --cov=[path]          measure coverage for filesystem path (multi-allowed)
...
```